source/blender/python/intern/bpy_interface.c

```diff
@@ -425,6 +425,14 @@ void BPY_python_use_system_env(void)
   py_use_system_env = true;
 }
 
+/* Game Engine transition */
+bool BPY_python_get_use_system_env(void)
+{
+  BLI_assert(!Py_IsInitialized());
```

Comment:
I tested in Debug mode on my computer and this assert error is always raised. Else audio and python work normally (but is was already working for me before). But if this is solving the issue for Daniel Er Bauer, I approve of course the pull request, but without the BLI_assert